jsapp/js/utils.es6

```diff
@@ -635,3 +635,18 @@ export function truncateFile(str, length) {
 
   return truncateString(truncatedString, length);
 }
+
+/**
+ * Generates a simple lowercase, underscored version of a string. Useful for
+ * quick filename generation
+ *
+ * @param {string} str
+ * @param {number} [startIndex=0]
+ * @param {number} [endIndex=str.length]
+ */
+export function generateAutoname(str, startIndex=0, endIndex=str.length) {
```

Comment:
Just being curious - does this match how `$autoname`s are generated? Maybe we could leave an url to backend code that makes that?

Reply:
I added a link to how the backend generates autoname for translations as that's the closest to "just a string" I could find in the file